content/flutter-configuration/built-in-variables.md

```diff
@@ -1,6 +1,7 @@
 ---
-description: Useful variables exported during builds
-title: Built-in environment variables
+description: Useful variables exported during builds using Flutter Workflow Editor
+title: Built-in environment variables using WFE
```

Comment:
I'd rather go for Flutter Workflow Editor instead of WFE (if the length of the title is still below 60 symbols), because Flutter is a relevant keyword, and WFE is not. Similarly in other places.